platform/core/src/services/UICustomizationService/UICustomizationService.ts

```diff
@@ -0,0 +1,273 @@
+import merge from 'lodash.merge';
+import { PubSubService } from '../_shared/pubSubServiceInterface';
+import CustomizationTypeInfo from './CustomizationTypeInfo';
+import { UICustomization, UIConfiguration, ExtensionManager } from './types';
+import { ComponentType } from 'react';
+
+const EVENTS = {
+  MODE_CUSTOMIZATION_MODIFIED: 'event::UICustomizationService:modeModified',
+  GLOBAL_CUSTOMIZATION_MODIFIED: 'event::UICustomizationService:globalModified',
+};
+
+type Obj = Record<string, unknown>;
+
+type ComponentReturn = {
+  component: ComponentType;
+  props?: Obj;
+};
+
+type NestedStrings = string[] | NestedStrings[];
```

Comment:
Can you move these to the CustomizationType file that you have

Reply:
Done